packages/phoenix-ng/projects/phoenix-ui-components/lib/components/ui-menu/ui-menu-wrapper/ui-menu-wrapper.component.spec.ts

```diff
@@ -1,13 +1,15 @@
 import { ComponentFixture, TestBed } from '@angular/core/testing';
 
 import { UiMenuWrapperComponent } from './ui-menu-wrapper.component';
+import { PhoenixUIModule } from '../../phoenix-ui.module';
 
 describe('UiMenuWrapperComponent', () => {
   let component: UiMenuWrapperComponent;
   let fixture: ComponentFixture<UiMenuWrapperComponent>;
 
   beforeEach(async () => {
     await TestBed.configureTestingModule({
+      imports: [PhoenixUIModule],
```

Comment:
If I don't add this, a bit of junk in the console logs

```
ngcc-jest-processor: running ngcc
  console.error
    NG0303: Can't bind to 'matTooltip' since it isn't a known property of 'button'.

      at logUnknownPropertyError (node_modules/@angular/core/fesm2020/core.mjs:10193:13)
      at elementPropertyInternal (node_modules/@angular/core/fesm2020/core.mjs:10065:17)
      at ɵɵproperty (node_modules/@angular/core/fesm2020/core.mjs:14497:9)
      at UiMenuWrapperComponent_Template (ng:/UiMenuWrapperComponent.js:30:9)
      at executeTemplate (node_modules/@angular/core/fesm2020/core.mjs:9647:9)
      at refreshView (node_modules/@angular/core/fesm2020/core.mjs:9510:13)
      at refreshComponent (node_modules/@angular/core/fesm2020/core.mjs:10707:13)
```